docs/source/reference/sensor_partitioning.rst

```diff
@@ -0,0 +1,84 @@
+Partitioning Sensors
+====================
+
+Often it is desirable to partition sensors across multiple sensor nodes. To this end
+StackStorm offers a few approaches to defining partition schemes.
+
+Each sensor node is identified by a name. The sensor nodename can be provided via a config
+property `sensor_node_name` as follows:
+
+::
+
+    [sensorcontainer]
+    ...
+    sensor_node_name = sensornode1
+
+
+1. Default
+~~~~~~~~~~
+
+In the default scheme all sensors are run on a particular node. As the name suggests it is
+default scheme which ships out of the box. It is useful in cases where there is a single
+sensor node.
+
+No change required in the config file but for complete-ness the config would be as follows:
+
+::
+
+    [sensorcontainer]
+    ...
+    sensor_node_name = sensornode1
```

Comment:
The value has no semantic meaning for us right now. So it could be `hostname-uuid` if the user chooses but no support from us really.